src/content/index.md

```diff
@@ -56,11 +56,13 @@ __page.html__
 </html>
 ```
 
-Then run `webpack` on the command-line to create `bundle.js`.
-</div>
 </div>
 
+Then run `webpack` on the command-line to create `bundle.js`.
 
 ## It's that simple
 
 __[Get Started](/guides/getting-started)__ quickly in our __Guides__ section, or dig into the __[Concepts](/concepts)__ section for more high-level information on the core notions behind behind webpack.
+
+</div>
+
```

Comment:
Why make this change? Was this part of resolving #1559? It seems this change actually might throw the layout off a bit.

Reply:
#1559 still not resolved, :( I thought it was processed, but dev mode made me confused, and above, I handle another UI issue in the content.